.mergify.yml

```diff
@@ -9,11 +9,15 @@ pull_request_rules:
       - label!=ci-waived
       - or:
         # Build test
-        - -check-success=build (gcc, 11, bfd, gcrypt)
-        - -check-success=build (gcc, 12, gold, openssl)
-        - -check-success=build (clang, 13, mold, gcrypt) 
-        - -check-success=build (clang, 14, lld, openssl)
-        - -check-success=build (clang, 15, bfd, auto)
+        # test build check only when specific files are changed ...
```

Comment:
Maybe "test build check only when the tests have been run" or something like that?

Reply:
Unfortunately, this is currently not possible. [Available conditions](https://docs.mergify.com/conditions/#attributes):

![Screenshot from 2023-04-27 18-24-53](https://user-images.githubusercontent.com/2879818/234927996-57aef685-2b41-48fb-8bee-a47e8ac949d7.png)

Reply:
Sorry, I misunderstood. I have updated the comments.